stylus/components/chip.styl

```diff
@@ -1,26 +1,58 @@
 @require 'settings/palette'
 @require 'tools/mixins'
 
-$chip-height = rem(40)
-$chip-font-size = rem(16)
+$chip-height-small = rem(32)
+$chip-font-size-small = rem(14)
+
+$chip-height-normal = rem(40)
+$chip-font-size-normal = rem(16)
 
 $chip
-    border-radius ($chip-height / 2)
-    height $chip-height
-    background var(--paleGrey)
-    padding (($chip-height - $chip-font-size) / 2)
     box-sizing border-box
     line-height 1
     display inline-flex
     align-items center
     margin-right rem(4)
     margin-bottom rem(4)
+    border-width 0
 
 $round-chip
-    width $chip-height
+    width $chip-height-normal
     text-align center
     justify-content center
 
+$small-size-chip
+    height $chip-height-small
+    padding (($chip-height-small - $chip-font-size-small) / 2)
+    border-radius ($chip-height-small / 2)
```

Comment:
You can put 50% for all the chips right ?

Reply:
Yeah, or even `100%`. Or anything above half the size, really.

Reply:
Actually with `50%` or `100%` we will end up with something like this: https://codesandbox.io/s/zealous-sun-9g03o

Reply:
OH yeah, sorry, you're absolutely right, the thing I said is for round object… 🤦‍♂

Reply:
Same mistake 😇